src/main/drivers/dshot_bitbang_decode.c

```diff
@@ -132,8 +130,8 @@ uint32_t decode_bb_bitband( uint16_t buffer[], uint32_t count, uint32_t bit)
         }
     }
 
-    startMargin = p - b;
-    DEBUG_SET(DEBUG_DSHOT_TELEMETRY_COUNTS, 3, startMargin);
+    const uint8_t startMargin = p - b;
```

Comment:
use `unsigned` if you want better code on ARM

Reply:
You mean as in `const unsigned int ...`?

Reply:
@ledvinap I changed each instance of uint8_t to `unsigned int`.  Is that what you'd recommend here?